config/sudoers-for-dev

```diff
@@ -18,31 +18,31 @@ Defaults	secure_path="/vx/code/config/admin-functions:/usr/local/sbin:/usr/local
-vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/expand-var-filesystem.sh
-vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/lockdown.sh
-vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/hash-signature.sh
-vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/generate-key.sh
-vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/setup-boot-entry.sh
-vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/create-machine-cert.sh
-vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/program-system-administrator-cards.sh
-vx-admin ALL=(root:ALL) NOPASSWD: /usr/local/bin/tpm2-totp
-vx-admin ALL=(root:ALL) NOPASSWD: /usr/sbin/reboot
-vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/mount-usb.sh
-vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/unmount-usb.sh
-vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/pactl.sh
-vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/intermediate-scripts/create-cert
+vx-vendor ALL=(root:ALL) NOPASSWD: /vx/vendor/vendor-functions/set-clock.sh
+vx-vendor ALL=(root:ALL) NOPASSWD: /vx/vendor/vendor-functions/rekey-via-tpm.sh
+vx-vendor ALL=(root:ALL) NOPASSWD: /vx/vendor/vendor-functions/expand-var-filesystem.sh
+vx-vendor ALL=(root:ALL) NOPASSWD: /vx/vendor/vendor-functions/lockdown.sh
+vx-vendor ALL=(root:ALL) NOPASSWD: /vx/vendor/vendor-functions/hash-signature.sh
+vx-vendor ALL=(root:ALL) NOPASSWD: /vx/vendor/vendor-functions/generate-key.sh
+vx-vendor ALL=(root:ALL) NOPASSWD: /vx/vendor/vendor-functions/setup-boot-entry.sh
+vx-vendor ALL=(root:ALL) NOPASSWD: /vx/vendor/vendor-functions/create-machine-cert.sh
+vx-vendor ALL=(root:ALL) NOPASSWD: /vx/vendor/vendor-functions/program-system-administrator-cards.sh
+vx-vendor ALL=(root:ALL) NOPASSWD: /usr/local/bin/tpm2-totp
+vx-vendor ALL=(root:ALL) NOPASSWD: /usr/sbin/reboot
+vx-vendor ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/mount-usb.sh
+vx-vendor ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/unmount-usb.sh
+vx-vendor ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/pactl.sh
+vx-vendor ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/intermediate-scripts/create-cert
 vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/intermediate-scripts/create-cert
 vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/intermediate-scripts/sign-message
 vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/backend/src/intermediate-scripts/*
 vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/scripts/*
-vx-services ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/hash-signature.sh
+vx-services ALL=(root:ALL) NOPASSWD: /vx/vendor/vendor-functions/hash-signature.sh
 vx-services ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/pactl.sh
-vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/config/admin-functions/timedatectl
+vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/config/vendor-functions/timedatectl
```

Comment:
@arsalansufi we do have the backend and the frontend using a couple of "vendor functions" but I think it's still not necessarily confusing. Or I could back out and rename the user `vx-admin` but rename the directory to `/vx/system` and `/vx/system-functions`. Either way I think it's valuable to get away from `admin`.

Reply:
Yeah thanks for flagging! Not too concerned about this either, as I actually want to change the control flow for these cases, moving the relevant function into vxsuite, with the vendor script calling the vxsuite script.

This is how we program sys admin cards in the vendor menu for example. The vendor menu / vxsuite-complete-system calls into vxsuite. I'm trying to avoid the other way around, vxsuite calling into vxsuite-complete-system, moving forward.

And will address these lingering examples at a future time, post cert submission.

Reply:
Speaking of which, we need to update https://github.com/votingworks/vxsuite/blob/f6ed8ec7c85b83bfff666f81dbfacb6d76df8d7d/libs/auth/src/signed_hash_validation.ts#L31, which I'll take care of

I have a hunch that `/vx/code/config/vendor-functions/timedatectl` is no longer used by `vx-ui`, i.e., kiosk-browser